### PR TITLE
Point towards `format_spec`; it is in other direction

### PR DESCRIPTION
**Pull request by tmiasko**

### CODE REVIEW

File: library/alloc/src/fmt.rs [hunk 1]
```diff
@@ -83,7 +83,7 @@
 //! # Formatting Parameters
 //!
 //! Each argument being formatted can be transformed by a number of formatting
-//! parameters (corresponding to `format_spec` in the syntax above). These
+//! parameters (corresponding to `format_spec` in the syntax below). These
```
Comment (joshtriplett) [inline library/alloc/src/fmt.rs:86]: Is there a way to link to an anchor on the same page? If so, how about changing this to just "syntax" and linking it?

**Review by joshtriplett — COMMENTED** [review]
I made one comment for potential improvement. If that improvement is possible, r=me with that included. If it isn't possible, then r=me for this change.

File: library/alloc/src/fmt.rs (hunk 1 above)
Comment (tmiasko) [reply library/alloc/src/fmt.rs]: Thanks for suggestion, replaced with a link to the syntax anchor.

**Pull request merged**